conf/modules.config

```diff
@@ -1,27 +1,55 @@
-/*
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Config file for defining DSL2 per module options and publishing paths
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Available keys to override module options:
-        ext.args   = Additional arguments appended to command in module.
-        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
-        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
-        ext.prefix = File name prefix for output files.
-----------------------------------------------------------------------------------------
-*/
-
-includeConfig './icgc-argo-workflows/modules.config'
-// includeConfig './nf-core/hisat2/align/module.config'
-// includeConfig './nf-core/samtools/sort/module.config'
-// includeConfig './nf-core/samtools/convert/module.config'
-// includeConfig './nf-core/bwamem2/mem/module.config'
-// includeConfig './nf-core/bwamem/module.config'
-// includeConfig './local/tar/module.config'
 process {
-    withName: 'NFCORE_VARIANTCALL:VARIANTCALL:PICARD_LIFTOVERVCF_SV' {
+    withName: 'PICARD_LIFTOVERVCF_SV' {
         ext.prefix = { "${meta.id}.short_variant" }
+        publishDir       = [
+            enabled : true,
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/liftover/short_variant/${meta.study_id}.${meta.donor_id}.${meta.id}"},
+            pattern: "{*lifted*}",
+            saveAs: { "${it}" }
+        ]
     }
-    withName: 'NFCORE_VARIANTCALL:VARIANTCALL:PICARD_LIFTOVERVCF_RA' {
+    withName: 'PICARD_LIFTOVERVCF_RA' {
         ext.prefix = { "${meta.id}.rearrangement" }
+        publishDir       = [
+            enabled : true,
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/liftover/rearrangement/${meta.study_id}.${meta.donor_id}.${meta.id}"},
+            pattern: "{*lifted*}",
+            saveAs: { "${it}" }
+        ]
+    }
+    withName: 'PAYLOAD_VARIANT_CALL_SV' {
+        publishDir       = [
+            enabled : true,
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/payload/short_variant/${meta.study_id}.${meta.donor_id}.${meta.id}"},
+            pattern: "{*payload.json,out/*}",
+            saveAs: { "${it}" }
+        ]
+    }
+    withName: 'PAYLOAD_VARIANT_CALL_RA' {
+        publishDir       = [
+            enabled : true,
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/payload/rearrangement/${meta.study_id}.${meta.donor_id}.${meta.id}"},
```

Comment:
updated all output paths from from `/payload/short_variant/SA626666` to `/payload/short_variant/MONSTAR-JP.DO111111.SA626666`

Reply:
@guanqiaofeng will `meta.ID` always be sampleID?